.github/workflows/ci.yml

```diff
@@ -78,6 +78,25 @@ jobs:
         with:
           key: ${{ github.job }}
 
+      - name: Run Zookeeper and Kafka
+        run: |
+          docker run \
+            --name sentry_zookeeper \
+            -d --network host \
+            -e ZOOKEEPER_CLIENT_PORT=2181 \
+            confluentinc/cp-zookeeper:6.2.0
+
+          docker run \
+            --name sentry_kafka \
+            -d --network host \
+            -e KAFKA_ZOOKEEPER_CONNECT=127.0.0.1:2181 \
+            -e KAFKA_LISTENERS=INTERNAL://0.0.0.0:9093,EXTERNAL://0.0.0.0:9092 \
+            -e KAFKA_ADVERTISED_LISTENERS=INTERNAL://127.0.0.1:9093,EXTERNAL://127.0.0.1:9092 \
+            -e KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT \
+            -e KAFKA_INTER_BROKER_LISTENER_NAME=INTERNAL \
+            -e KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1 \
+            confluentinc/cp-kafka:6.2.0
+
```

Comment:
Can we have this in a docker-compose or just anything that makes it easy to run it locally without copy-pasting it into your command line?

Reply:
Currently the best way to run it locally for Sentry developers is with Sentry devservices. It should just work. This is the same as what Snuba + Python Arroyo does. At this point, I would avoid adding something separate for those not running devservices.